mmv1/templates/terraform/decoders/apigee_organization_keyvaluemaps.go.tmpl

```diff
@@ -0,0 +1,10 @@
+config := meta.(*transport_tpg.Config)
```

Comment:
Since this decoder is exactly the same as `apigee_environment_keyvaluemaps.go.tmpl`, it might be better to reuse it and rename it to `apigee_keyvaluemaps.go.tmpl`.

Reply:
agreed, file changed. thanks